src/students/entities/student-grades.entity.ts

```diff
@@ -0,0 +1,27 @@
+import { BaseEntity, Column, Entity, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+import { Student } from './student.entity';
+
+@Entity('student_grades')
+export class StudentGrades extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ type: 'float', nullable: false })
```

Comment:
Czy nie lepiej byłoby dać type: 'tinyint'? łatwiej będzie z animacją gwiazdek.

Reply:
mamy coś takiego jak `projectDegree` - Ocena zadania zaliczeniowego w kursie w zakresie 0 - 5. 
Rozumiem, że to są to nasze oceny za projekt końcowy, a tam mamy zmiennoprzecinkowe, więc `float`.